src/Windowing/Silk.NET.Windowing.Common/Structs/WindowIcon.cs

```diff
@@ -3,7 +3,7 @@
 // You may modify and distribute Silk.NET under the terms
 // of the MIT license. See the LICENSE file for details.
 
-namespace Silk.NET.Windowing.Common.Structs
+namespace Silk.NET.Windowing.Structs
```

Comment:
Imo it would make sense to remove the special `Structs` namespace.

Reply:
This shouldn't even exist, ReSharper acting up. Will lookover this shortly.

Reply:
Fixed.